PocketKit/Sources/PocketKit/Home/HomeViewModel.swift

```diff
@@ -131,7 +131,7 @@ class HomeViewModel {
 
         networkPathMonitor.updateHandler = { [weak self] path in
             if path.status == .satisfied {
-                self?.refresh { }
+                self?.refresh(isForced: true) { }
```

Comment:
I missed this during the home refresh work - I believe this should be a force refresh since it should only occur when network connectivity change